v2.6/getting-started/kubernetes/installation/hosted/canal/index.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Canal Hosted Install
+canonical_url: 'https://docs.projectcalico.org/v3.0/getting-started/kubernetes/installation/hosted/kubeadm/'
+---
+
+## About Canal
+
+Canal allow users to easily deploy Calico and flannel networking
+together as a unified networking solution - combining Calico’s
+industry-leading network policy enforcement with the flannel overlay and
+non-overlay network connectivity options.
+
+Note that the Canal currently uses the Calico and flannel projects as is with
```

Comment:
What I think would probably be helpful here is to make flannel into a link which allows them to easily visit the flannel gitHub repo if they are curious to find out more about flannel.